README.md

```diff
@@ -1,2 +1,196 @@
-# Catalog
-For now  lets keep it null
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🧪 Linters](#linters)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog <a name="about-project"></a>
+
+Catalog is ruby based with terminal user interface project in which you can organize you things accroding to verious item object such as book, music, video etc
+
```

Comment:
- [OPTIONAL] I think it would be best to avoid typos in your README file because this file summarizes the project and everyone who visits your repo will start by reading what is in the README file to get an idea of your project 🙃 🙃. A perfect README file makes us more professional 😎 😉